control/tap-ctl-stats.c

```diff
@@ -105,11 +105,17 @@ tap_ctl_stats_fwrite(pid_t pid, int minor, FILE *stream)
 
 	prot  = PROT_READ|PROT_WRITE;
 	flags = MAP_ANONYMOUS|MAP_PRIVATE;
-	bufsz = sysconf(_SC_PAGE_SIZE);
+
+	err = sysconf(_SC_PAGE_SIZE);
+	if (err < 1) {
```

Comment:
sysconf signals errors with -1 and that it was we are looking for.
The fact that, for this specific value we cannot get 0 is just a happy coincidence. If you are so sure that it is impossible to get 0, even better, no need to bother.

Check the standard return code and that is -1.